workalendar/europe/latvia.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+from datetime import date
+from workalendar.core import WesternCalendar, ChristianMixin
+
+
+class Latvia(WesternCalendar, ChristianMixin):
+    "Latvia"
+
+    FIXED_HOLIDAYS = WesternCalendar.FIXED_HOLIDAYS + (
+        (5,  1,  "Labour Day"),
+        (6,  23, "Midsummer Day"),
+        (6,  24, "St. John's Day"),
+        (11, 18, "Proclamation Day"),
+        (12, 31, "New Years Eve"),
+    )
+
+    include_good_friday = True
+    include_easter_monday = True
+    include_christmas_eve = True
+    include_christmas = True
+    include_boxing_day = True
+
+    def get_independence_days(self, year):
+        """returns a possibly empty list of (date, holiday_name) tuples"""
+        days = []
+        if year > 2004:
+            actual_date = date(year, 5, 4)
+            days = [(actual_date, "Restoration of Independence Day")]
+            if actual_date.weekday() in self.get_weekend_days():
+                days += [(self.find_following_working_day(actual_date),
```

Comment:
It's returning a list that might be empty. days.extend() would work though.
~~~
>>> x = ['a','b','c']
>>> y = []
>>> x.append(y)
>>> x
['a', 'b', 'c', []]
>>> x.pop()
[]
>>> x
['a', 'b', 'c']
>>> x += []
>>> x
['a', 'b', 'c']
>>> x.extend([])
>>> x
['a', 'b', 'c']
~~~